ktxspec.adoc

```diff
@@ -1224,9 +1238,35 @@ level. The offset of a level's `levelImages` is provided by the
 When `<<_supercompressionscheme,supercompressionScheme>> != 0` these
 bytes are formatted as specified in the scheme documentation.
 
-=== mipPadding
-`mipPadding` is between 0 and 7 bytes of value `0x00` to make sure that all
-mip level data starts at a file offset that is a multiple of 8.
+==== mipPadding
+`mipPadding` is between 0 and stem:[lcm(texel\ block\ size, 4) -
```

Comment:
What does "between" mean here? It reads as if it's optional but I'm assuming that it's required to be the specified number of 0 bytes.

Reply:
... Oh, sorry, I get it now. Ignore this.